indra/newview/llappviewer.cpp

```diff
@@ -547,7 +547,7 @@ static void settings_to_globals()
     LLRender::sGLCoreProfile = gSavedSettings.getBOOL("RenderGLContextCoreProfile");
 #endif
     LLRender::sNsightDebugSupport = gSavedSettings.getBOOL("RenderNsightDebugSupport");
-    LLImageGL::sGlobalUseAnisotropic    = gSavedSettings.getBOOL("RenderAnisotropic");
+    LLRender::sAnisotropicFilteringLevel = static_cast<F32>(gSavedSettings.getU32("RenderAnisotropicLevel"));
```

Comment:
Does sAnisotropicFilteringLevel really need to be an F32?  I assume we need to cast somewhere down the line anyways - if so then I suppose this is fine.